frontend/src/main/utils/users.js

```diff
@@ -1,17 +1,48 @@
 import { useQuery } from "react-query";
 import axios from "axios";
+import { useBackendMutation } from "./useBackend";
 
 export function useUsers() {
-  return useQuery("users", async () => {
-    const uri = "/api/admin/users";
-    try {
-      const response = await axios.get(uri);      
-      return response.data ;
-    } catch (e) {
-      console.error(`Error getting data from ${uri}:`,e);
-      return [];
+  return useQuery(
+    "users",
+    async () => {
+      const uri = "/api/admin/users";
+      try {
+        const response = await axios.get(uri);
+        return response.data;
+      } catch (e) {
+        console.error(`Error getting data from ${uri}:`, e);
+        return [];
+      }
+    },
+    {
+      initialData: [],
     }
-  }, {
-    initialData: []
-  });
+  );
+}
+
+export function useSuspendUser() {
+  return useBackendMutation(
+    (cell) => ({
+      method: "post",
+      url: "/api/admin/users/suspend",
+      params: { userId: cell.row.values.id },
+    }),
+    {},
+    // Stryker disable next-line all : hard to set up test for query key invalidation for mutations
```

Comment:
When I Ctrl+F'd the codebase for other examples that used useBackendMutation, I noticed they all disable checking for the "users" query key invalidation because it was hard to simulate / test.